playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerView.java

```diff
@@ -474,16 +474,16 @@ public List<Cue> getModifiedSubtitlePosition(List<Cue> cueList, PKSubtitlePositi
                     newCueList.add(cue);
                     continue;
                 }
+
```

Comment:
if ((cue.line !=  Cue.DIMEN_UNSET || cue.position != Cue.DIMEN_UNSET)
why this condition is still required.

Reply:
Removed this condition.